components/sst_importer/src/sst_importer.rs

```diff
@@ -472,16 +472,30 @@ impl<E: KvEngine> SSTWriter<E> {
         let commit_ts = TimeStamp::new(batch.get_commit_ts());
         for m in batch.get_pairs().iter() {
             let k = Key::from_raw(m.get_key()).append_ts(commit_ts);
-            self.put(k.as_encoded(), m.get_value())?;
+            let is_delete = {
+                #[cfg(feature = "prost-codec")]
+                {
+                    m.get_op() == pair::Op::Delete
+                }
+                #[cfg(not(feature = "prost-codec"))]
+                {
+                    m.get_op() == PairOp::Delete
+                }
+            };
```

Comment:
Please move it to `fn put` to avoid flag arguments.

Reply:
Done